src/memory.h

```diff
@@ -6,6 +6,7 @@
 #include "macros.h"
 
 #define RAM_END 0x80400000
+#define EXTENDED_RAM_END 0x80600000
```

Comment:
```suggestion
#define EXTENDED_RAM_END 0x80800000
```

Maybe change this to 8MB just for the sake of easy modding, if this isn't actually part of matching?

Reply:
Sure. Since it doesn't affect the code, we don't need to match what works for JFG, and might as well make it work for 8MB instead. Done.